src/App.js

```diff
@@ -1,27 +1,17 @@
-import logo from './logo.svg';
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable **all** linter checks on your code to ensure that your code is quality and robust.